src/VocabGeneration.test.js

```diff
@@ -231,12 +231,32 @@ describe("Suite for generating common vocabularies (marked as [skip] to prevent
   // it("tests a single custom vocab", async () => {
   it.skip("tests a single custom vocab", async () => {
     await generateVocabArtifact({
+      inputResources: ["http://www.w3.org/2006/03/test-description#"],
+      nameAndPrefixOverride: "td",
+      descriptionFallback: "Test description",
+
+      // inputResources: ["https://protect.oeg.fi.upm.es/def/gdprif#"],
+      // inputResources: ["https://protect.oeg.fi.upm.es/def/gdprif/ontology.ttl"],
+      // ignoreNonVocabTerms: true,
+
+      // inputResources: ["http://purl.org/dc/dcam/"],
+      // nameAndPrefixOverride: "dcam",
+
       // inputResources: ["https://w3id.org/plasma#"],
       // inputResources: ["https://w3id.org/oac#"],
 
-      inputResources: [
-        "https://raw.githubusercontent.com/coolharsh55/plasma/main/plasma.ttl",
-      ],
+      // inputResources: ["http://www.w3.org/ns/dx/prof/"],
+      // inputResources: ["https://www.w3.org/TR/dx-prof/rdf/prof.ttl"],
+      // nameAndPrefixOverride: "prof",
+      // ignoreNonVocabTerms: true,
+
+
+      // inputResources: [
+      //   "https://protect.oeg.fi.upm.es/odrl-access-control-profile/oac.ttl",
+      // ],
+      // inputResources: [
+      //   "https://raw.githubusercontent.com/coolharsh55/plasma/main/plasma.ttl",
+      // ],
 
       // inputResources: ["http://rdf-vocabulary.ddialliance.org/xkos#"],
```

Comment:
Is all this commented out code supposed to get into versioning?

Reply:
Yeah, I just keep it there for when I revisit vocabs, which can evolve over time (in terms the conneg, or needing new overrides, etc.). I don't see a big issue with it - it's kinda like an archive, or a history of vocabs I've investigated and usually (but not always) added to Inrupt's bundled 'Common RDF Vocabs' artifact.
I've no big issue with removing it either, but I do find I revisit vocabs often enough to justify keeping it.

Reply:
I would either have these as proper tests individually running on vocabularies we want to support over time, or not be present at in the codebase.